### PR TITLE
Bump to 0.27.4 and update tests

### PR DESCRIPTION
**Pull request by paulinebessoles**
This PR aims to bump the module to 0.27.4 Decidim version, and update tests according to https://github.com/decidim/decidim/pull/11042

It also fixes the issues with admin password updates

### CODE REVIEW

**Comment by armandfardeau** [comment]
Hello thanks for your contribution !

Could you please fix the failing CI workflow?

**Review by paulinebessoles — COMMENTED** [review]
Works great!

**Review by armandfardeau — APPROVED** [review]

File: lib/decidim/friendly_signup/version.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 module Decidim
   # This holds the decidim-meetings version.
   module FriendlySignup
-    DECIDIM_VERSION = "0.27.0"
+    DECIDIM_VERSION = "0.27.4"
     COMPAT_DECIDIM_VERSION = "~> 0.27"
     VERSION = "0.4.4"
```
Comment (Quentinchampenois) [inline lib/decidim/friendly_signup/version.rb]: We should bump the `VERSION` at least to 0.4.5 because we are changing dependencies

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
We should update the current module's version.

Also we could add the release generator and push to rubygems job in CD

File: lib/decidim/friendly_signup/version.rb (hunk 1 above)
Comment (armandfardeau) [reply lib/decidim/friendly_signup/version.rb]: Very true

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -110,3 +110,45 @@ jobs:
...
+      - name: Publish to RubyGems
+        run: |
+          mkdir -p $HOME/.gem
+          touch $HOME/.gem/credentials
+          chmod 0600 $HOME/.gem/credentials
+          printf -- "---\n:rubygems_api_key: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
+          gem build *.gemspec
+          gem push *.gem
+        env:
+          GEM_HOST_API_KEY: "${{secrets.RUBYGEMS_API_KEY}}"
```
Comment (armandfardeau) [inline .github/workflows/ci.yml:154]: I think you can use the following:
```yaml
- name: Publish gem
  uses: OpenSourcePolitics/decidim-publish-gem-action@master
  with:
    rubygems_api_key: ${{ secrets.RUBYGEMS_API_KEY }}
    github_token: ${{ secrets.GITHUB_TOKEN }}

**Review by armandfardeau — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,154 @@
+name: "[CI] Test"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```
Comment (armandfardeau) [inline .github/workflows/ci.yml]: Wouldn't it be easier to just do?

```suggestion
on: "push"
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      - name: Publish to RubyGems
+        run: |
+          mkdir -p $HOME/.gem
+          touch $HOME/.gem/credentials
+          chmod 0600 $HOME/.gem/credentials
+          printf -- "---\n:rubygems_api_key: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
+          gem build *.gemspec
+          gem push *.gem
+        env:
+          GEM_HOST_API_KEY: "${{secrets.RUBYGEMS_API_KEY}}"
```
Comment (armandfardeau) [inline .github/workflows/ci.yml]: You can use our custom action here:

```suggestion
      - name: Publish gem
         uses: OpenSourcePolitics/decidim-publish-gem-action@master
         with:
           rubygems_api_key: ${{ secrets.RUBYGEMS_API_KEY }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
```

**Review by armandfardeau — CHANGES_REQUESTED** [review]

**Pull request merged**